GADDirectory/GADQuery.h

```diff
@@ -0,0 +1,46 @@
+//
+//  GADQuery.h
+//  DB
+//
+//  Created by Liam li on 3/5/17.
+//  Copyright © 2017 AppDev. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "GADPerson.h"
+
+@interface GADQuery : NSObject
+
+@property (nullable,nonatomic, strong) NSString *lastName;
+@property (nullable,nonatomic, strong) NSString *firstName;
+@property (nullable,nonatomic, strong) NSString *campusAddressOrPOBox;
+@property (nullable,nonatomic, strong) NSString *FacStaffDeptOffice;
+@property (nullable,nonatomic, strong) NSString *studentMajor;
+@property (nullable,nonatomic, strong) NSString *hiatus;
+@property (nullable,nonatomic, strong) NSString *computerUsername;
+@property (nullable,nonatomic, strong) NSString *campusPhone;
+@property (nullable,nonatomic, strong) NSString *homeAddress;
+@property (nullable,nonatomic, strong) NSString *SGA;
+@property (nullable,nonatomic, strong) NSString *concentration;
+@property (nullable,nonatomic, strong) NSString *studentClass;
+
+-(void)setlastName: (nonnull NSString*) lastName;
```

Comment:
All of these set functions are redundant; an API consumer may use `-[GADQuery setLastName:]` which is automatically generated by the `@property` statement. For an initialized query object `GADQuery *q`, they also may do `q.lastName = ""` automatically. You can remove these functions.